ssas/service/admin/api.go

```diff
@@ -178,6 +179,52 @@ func updateGroup(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getSystem(w http.ResponseWriter, r *http.Request) {
+	id := chi.URLParam(r, "id")
+	trackingID := ssas.RandomHexID()
+	systemEvent := ssas.Event{Op: "UpdateSystem", TrackingID: trackingID}
+
+	s, err := ssas.GetSystemByID(id)
+	if err != nil {
+		systemEvent.Help = fmt.Sprintf("could not find system %s", id)
+		ssas.OperationFailed(systemEvent)
+		jsonError(w, http.StatusNotFound, fmt.Sprintf("could not find system %s", id))
+		return
+	}
+
```

Comment:
We probably ought to add a verification that this system is active

Reply:
dobe

Reply:
Can we also add a unit test case?